setup/sql/5_4_hotfixes.php

```diff
@@ -17,4 +17,19 @@
 ), array(
 	"1" => array("integer", 1)
 ));
+?>
+<#4>
+<?php
+if ($ilDB->tableExists('license_data')) {
+	$ilDB->dropTable('license_data');
+}
+?>
+<#5>
+<?php
+$settings = new ilSetting('license');
```

Comment:
A "DELETE FROM settings WHERE module = ".$ilDB->quote('license', "text");
would reduce dependencies to non setup classes.

Reply:
@smeyer-ilias I will change this, thx!